src/models/movie.js

```diff
@@ -0,0 +1,114 @@
+// const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+// const getMoviesData = async () => {
+//   const movies = await prisma.movie.findMany();
+//   return {movies}
+// };
+
+const getMoviesData = async (runtimeLt, runtimeGt) => {
+  if (runtimeLt !== undefined && runtimeGt !== undefined) {
+    const movies = await prisma.movie.findMany({
+      where: {
+        OR: [
+          {
+            runtimeMins: {
+              lt: Number(runtimeLt),
+            },
+          },
+          {
+            runtimeMins: {
+              gt: Number(runtimeGt),
+            },
+          },
+        ],
+      },
+      include: {
+        screenings: true,
+      },
+    });
+    console.log("both");
+    return { movies };
+  } else if (runtimeLt !== undefined) {
+    const movies = await prisma.movie.findMany({
+      where: {
+        runtimeMins: {
+          lt: Number(runtimeLt),
+        },
+      },
+      include: {
+        screenings: true,
+      },
+    });
+    console.log("less than");
+    return { movies };
+  } else if (runtimeGt !== undefined) {
+    const movies = await prisma.movie.findMany({
+      where: {
+        runtimeMins: {
+          gt: Number(runtimeGt),
+        },
+      },
+      include: {
+        screenings: true,
+      },
+    });
+    console.log("greater than");
+    return { movies };
+  } else {
+    const movies = await prisma.movie.findMany({
+      include: {
+        screenings: true,
+      },
+    });
+    console.log("neither");
+    return { movies };
+  }
+};
```

Comment:
Nice logic for handling combinations of run times being provided! Could using a switch statement be useful here for handling the different parameters you want passing in depending on the timings provided?